wix/main.wxs

```diff
@@ -0,0 +1,222 @@
+<?xml version='1.0' encoding='windows-1252'?>
+<!--
+  Copyright (C) 2017 Christopher R. Field.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+
+<!--
+  Please do not remove these pre-processor If-Else blocks. These are used with
+  the `cargo wix` subcommand to automatically determine the installation
+  destination for 32-bit versus 64-bit installers. Removal of these lines will
+  cause installation errors.
+-->
+<?if $(var.Platform) = x64 ?>
+    <?define Win64 = "yes" ?>
+    <?define PlatformProgramFilesFolder = "ProgramFiles64Folder" ?>
+<?else ?>
+  <?define Win64 = "no" ?>
+  <?define PlatformProgramFilesFolder = "ProgramFilesFolder" ?>
+<?endif ?>
+
+<Wix xmlns='http://schemas.microsoft.com/wix/2006/wi'>
+
+    <Product
+        Id='*'
+        Name='Notion'
+        UpgradeCode='2AC3458D-A24D-4828-AF47-013B30999B54'
+        Manufacturer='The Notion Maintainers'
+        Language='1033'
+        Codepage='1252'
+        Version='$(var.Version)'>
+
+        <Package Id='*'
+            Keywords='Installer'
+            Description='The Hassle-Free JavaScript Toolchain Manager'
+            Manufacturer='The Notion Maintainers'
+            InstallerVersion='450'
+            Languages='1033'
+            Compressed='yes'
+            InstallScope='perMachine'
+            SummaryCodepage='1252'
+            Platform='$(var.Platform)'/>
+
+        <MajorUpgrade
+            Schedule='afterInstallInitialize'
+            DowngradeErrorMessage='A newer version of [ProductName] is already installed. Setup will now exit.'/>
+
+        <Media Id='1' Cabinet='media1.cab' EmbedCab='yes' DiskPrompt='CD-ROM #1'/>
+        <Property Id='DiskPrompt' Value='Notion Installation'/>

```

Comment:
I don't think you need neither the `DiskPrompt` attribute on the Media element nor the `DiskPrompt` property...
Any specific reason you added that? 

There's a `CompressionLevel` attribute on the Media element, unfortunately it's not documented too well: http://wixtoolset.org/documentation/manual/v3/xsd/wix/simple_type_compressionleveltype.html

I use `high` in my project and it works fine, maybe you wanna try and compare installer sizes?

Reply:
Those were added automatically by the `cargo wix` bootstrap process. I thought they were odd since we aren't using a collection of CD-ROMs any more. If they can be removed without trouble I'd definitely prefer to do so, I'll look into it.

Thanks for the tip on the `CompressionLevel`, I'll try out `high` and see how it does 👍